.github/workflows/arena-brazil-testing-deploy.yml

```diff
@@ -17,8 +17,8 @@ jobs:
     name: Build and deploy to Brazil testing
     runs-on: ubuntu-latest
     environment:
-      name: testing
-      url: https://brazil-testing.curseofmirra.com/board
+      name: testing-brazil
+      url: https://arena-brazil-testing.curseofmirra.com/
```

Comment:
wont this break the unity client? i think they're using "brazil-testing.curseofmirra.com"

Reply:
Nice catch! I'll open a PR fixing that.